src/index.html

```diff
@@ -5,8 +5,427 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav>
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#apple" class="header__link header__link--is-active is-active">Apple</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#samsung" class="header__link">Samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#smarphones" class="header__link">Smartphones</a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#laptops"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#gadgets" class="header__link">Gadgets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#tablets" class="header__link">Tablets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#photo" class="header__link">Photo</a>
+          </li>
+
+          <li class="header__list">
+            <a href="#video" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+    <main class="product">
+      <section class="product__container">
+        <article
+          action="#card"
+          method="post"
+          class="card product__card"
+          data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="Product card image"
+            class="card__image"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">
+            Prodect code: 195434
+          </p>
+          <div class="card__score">
+            <div class="card__stars card__stars--4">
```

Comment:
https://github.com/mate-academy/layout_catalog/blob/master/checklist.md
![image](https://user-images.githubusercontent.com/91411692/223364778-53b2a76b-7e5d-4d63-86f6-3c419a1657d5.png)

```suggestion
          <img
            src="images/imac.jpeg"
            alt="Product card image"
            class="card__image"
          >
          
          <h2 class="card__title">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h2>
          
          <p class="card__code">
            Prodect code: 195434
          </p>
          
          <div class="card__score">
            <div class="card__stars card__stars--4">
```